src/app/login/login/login.component.html

```diff
@@ -1,13 +1,9 @@
-<link href="//netdna.bootstrapcdn.com/bootstrap/3.0.0/css/bootstrap.min.css" rel="stylesheet" id="bootstrap-css">
-<script src="//netdna.bootstrapcdn.com/bootstrap/3.0.0/js/bootstrap.min.js"></script>
-<script src="//code.jquery.com/jquery-1.11.1.min.js"></script>
-
 <div class="container input-group">
   <div class="row">
     <div class="col-sm-6 col-md-4 col-md-offset-4">
       <h1 class="text-center login-title">Sign in to Your task list</h1>
       <div class="account-wall">
-        <img class="profile-img" src="./../../../assets/images/photo.png"
+        <img class="profile-img" src="../../../assets/Images/photo.png"
```

Comment:
Please rename folder to image the do the same with a real folder in assets

Reply:
it is already renamed